WG-TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+# OCI Working Group Template
+
+The following is a template that the TOB and/or proposers of a new OCI Working Group may use to document the intended scope and activities of the OCI Working Group following its establishment. {Bracketed sections below} should be filled in with the Working Group's specific details.
+
+# {WG NAME} OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the {WG NAME} Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+{Briefly describe the purpose of the WG.}
+
+## Scope
+
+* {Describe the activities and work that are in scope for the WG.}
+
+## Out of Scope
+
+* {Where helpful for clarity or to avoid confusion, describe any activities or topics that are out of scope for the WG.}
+
+## Intended work product
+
+* {Describe the intended work product and outputs of the WG, particularly indicating whether they consist of specification(s) and/or other project(s).}
+
+## Governance
+
+* **Working Group**:
+  * The TOB is establishing the WG as an OCI Working Group, pursuant to [section 6(p)](https://github.com/opencontainers/tob/blob/master/CHARTER.md#6-technical-oversight-board-tob) of the OCI Charter.
+* **Owners**:
+  * The WG proposal to the TOB will specify one or more initial "owners" of the WG.
```

Comment:
Should we have a minimum of 2 or more owners? If something is to be run under OCI, seems it should be big enough, or interesting enough for multiple people to be engaged, to also assure the working group can scale to the needs of it being so visible under OCI.

Reply:
+1 here - from multiple organizations would be great too.

Reply:
Agreed, two or more from at least 2 organizations sounds good.